torch/_inductor/scheduler.py

```diff
@@ -1137,15 +1137,21 @@ def get_consumer_subnode_for(
     def get_producer_subnode_for(
         self, consumer: BaseSchedulerNode
     ) -> Optional[BaseSchedulerNode]:
+        producers = []
```

Comment:
I think ideally:

a) As of https://developer.nvidia.com/blog/cuda-12-1-supports-large-kernel-parameters/ i think we can include > 256 arguments so no longer need this longer (can check cuda version at runtime)

b) we could split up the >256 foreach nodes during scheduling time to avoid breaking this fusion. 

Definitely a) would be easier & faster. But i know there is a sev so i'm not opposed to landing this.

Reply:
Triton had strange errors last time I tried this, so there might be non-cuda limitations as well. I will try this though (in a separate PR)